mq/main/mq-broker/broker-core/src/main/java/com/sun/messaging/jmq/jmsserver/Broker.java

```diff
@@ -1233,15 +1231,21 @@ private int _start(boolean inProcess, Properties propsFromCommandLine,
                 if (!(ex instanceof LoopbackAddressException)) {
                     logger.logStack(Logger.INFO,
                     BrokerResources.X_INTERNAL_EXCEPTION, ex.getMessage(), ex);
+                } else {
+                    // WORKAROUND - Payara FISH-642
+                    // Check if we've already destroyed the broker before attempting to fall back to non-clustered
+                    // Can't touch Globals here since we may have already wiped the global config - touching it
+                    // will potentially blow away pre-existing root logging config
+                    if (exited) {
+                        logger.log(Logger.ERROR, ex.getMessage());
+                        if (failStartThrowable != null) {
+                            failStartThrowable.initCause(new Exception(ex));
+                        }
+                        return (1);
```

Comment:
minor formatting - return is not a method call (remove parens)

Reply:
Was copied from above, but yes should correct 👍